pic-domain/src/main/kotlin/com/mashup/pic/domain/common/RedisConfig.kt

```diff
@@ -0,0 +1,46 @@
+package com.mashup.pic.domain.common
+
+import org.springframework.cache.CacheManager
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.data.redis.cache.RedisCacheConfiguration
+import org.springframework.data.redis.cache.RedisCacheManager
+import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory
+import org.springframework.data.redis.core.RedisTemplate
+import org.springframework.data.redis.serializer.GenericJackson2JsonRedisSerializer
+import org.springframework.data.redis.serializer.RedisSerializationContext
+import org.springframework.data.redis.serializer.StringRedisSerializer
+import java.time.Duration
+
+
+@Configuration
+class RedisConfig {
+
+    @Bean
+    fun redisTemplate(): RedisTemplate<*, *> {
+        val redisTemplate = RedisTemplate<Any, Any>()
+        redisTemplate.setConnectionFactory(lettuceConnectionFactory())
+        redisTemplate.keySerializer = StringRedisSerializer()
+        redisTemplate.hashKeySerializer = StringRedisSerializer()
+        redisTemplate.valueSerializer = GenericJackson2JsonRedisSerializer()
+        return redisTemplate
```

Comment:
```suggestion
        return RedisTemplate<Any, Any>().apply { 
            connectionFactory = lettuceConnectionFactory()
            keySerializer = StringRedisSerializer()
            hashKeySerializer = StringRedisSerializer()
            valueSerializer = GenericJackson2JsonRedisSerializer()
        }
```
이럴때 쓰기 좋은 Scope function이 있어요.
그리고 set~~는 자바 스타일이에요. 왠만하면 value = ~~` 로 작성하시는게 보기 좋습니다~